src/common/plugin.ts

```diff
@@ -47,7 +55,32 @@ export abstract class Plugin implements IPlugin {
 
     const response = this.container.channelService.hasPermission(channelName, this.permission);
     if (!response) {
-      message.reply(`Please use this command in a \`${this.permission}\` channel`);
+      const rooms = Object.values(Constants.Channels[this.permission]);
+      let addonText = '';
+
+      //if this fails it simply means that a channel the bot expected to exist didn't
+      try {
+        if (rooms.length === 0 && this.permission.toString() != 'Private') {
+          addonText = 'their are no permanant channels oh this type declared.';
+        } else if (rooms.length === 1) {
+          const id = this.container.guildService.getChannel(rooms[0]).id;
+          addonText = `, <#${id}> is the only channel whith this type`;
+        } else {
+          const id = [
+            this.container.guildService.getChannel(rooms[0]).id,
+            this.container.guildService.getChannel(rooms[1]).id,
+          ];
+          if (rooms.length == 2) {
+            addonText = `, <#${id[0]}> and <#${id[1]}> are the only two with this channel type`;
+          } else {
+            addonText = `, <#${id[0]}> and <#${id[1]}> are two of the channels this channel type`;
```

Comment:
would rather do a slice on `rooms` that caps the length at <=3, and then do a map&join

Reply:
don't think the join is worth it, did the rest though.

Reply:
> I like the idea
> ![image](https://user-images.githubusercontent.com/39809146/112398583-ebb1f680-8cda-11eb-96a1-0fffdec25773.png)
> Only issue is that there are some channels that are hidden to the user, but it'll still create a link for it. Only `#verify` afaik, but it should check to make sure that the user has the perms for that channel
> 
> This is what it would look like to a user without the perms
> ![image](https://user-images.githubusercontent.com/39809146/112398996-c4a7f480-8cdb-11eb-9efa-cec64d165c5a.png)

Now checks to see if the user is aware of that channel.
![image](https://user-images.githubusercontent.com/10034057/112403683-3c2e5180-8ce5-11eb-8e6a-0cdd1701566b.png)